community/julia/APKBUILD

```diff
@@ -2,10 +2,10 @@
 # Contributor: Shiz <hi@shiz.me>
 # Maintainer: Shiz <hi@shiz.me>
 pkgname=julia
-pkgver=0.6.0
+pkgver=0.6.1
 # Keep in sync with deps/libuv.version.
 _libuv_ver=52d72a52cc7ccd570929990f010ed16e2ec604c8
-_llvm_ver=3.9
+_llvm_ver=4
```

Comment:
Julia still recommends LLVM 3.9 according to [README](https://github.com/JuliaLang/julia/blob/v0.6.1/README.md). They are quite sensitive to specific LLVM version, so I’d rather stick to 3.9 until they decide to update.

/cc @yuyichao